### PR TITLE
Take selected version from url

### PR DESCRIPTION
**Pull request by Philippus**

### CODE REVIEW

**Review by eed3si9n — APPROVED** [review]

**Comment by Philippus** [comment]
I should also make a PR for the 0.13.x branch I guess?

**Comment by eed3si9n** [comment]
I think nanoc stuff is only active on 1.x

**Comment by Philippus** [comment]
I've made another PR for 0.13 https://github.com/sbt/website/pull/542
(the referenced issue was about the 0.13 page)

File: src/nanoc/content/set-versions.js [hunk 1]
```diff
@@ -6,12 +6,8 @@ $(document).ready(function() {
    var snapshotPath = 'snapshot/';
    var docsPath = "docs/";
 
-   // get the version drop-down
-   var select = $("#versions");
-   // the currently selected value is the current version
-   var selected = select.val();
-   // clear the options, which should only include the current version
-   select.html('');
```
Comment (eed3si9n) [inline src/nanoc/content/set-versions.js:14]: I'm now getting 

```
Uncaught ReferenceError: select is not defined
    at HTMLDocument.<anonymous> (set-versions.js:19)
    at Object.resolveWith (jquery-1.6.2.min.js:16)
    at Function.ready (jquery-1.6.2.min.js:16)
    at HTMLDocument.B (jquery-1.6.2.min.js:16)
```

at

```
select.append('<option ' + sel + 'value="' + v + '">' + label + '</option>');
```

**Pull request merged**